sbt-exercise/src/sbt-test/sbt-exercise/scalariform/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.2.8
```

Comment:
Same here, let's remove all the build.properties files within the scripted tests